Add node.js migration guide (#3040)
wasp-lang/wasp: web/docs/project/build-preview.md

---
title: Production Build Preview
---

> The `wasp build start` command allows you to preview your production build locally before deploying. This is essential for testing how your app behaves in production conditions and ensuring everything works correctly with your production configuration.

Reviewer: Essential and ensure are strong words.
I would tone it down a bit.

```suggestion
The `wasp build start` command allows you to preview your production build locally before deploying. This helps you test how your app behaves in production conditions and verify that it works correctly with your production configuration.
```
